### ISSUE TITLE
Right joycon pinout? 

### ISSUE THREAD

**Issue by SamanthaRenamon** [dekuNukem/Nintendo_Switch_Reverse_Engineering#87]
Do you have the schematics of the right joycon pinout i need to make a bridge for the r button

**Comment by ratijas** [comment]
Yeah, need to fix the R button myself. R button box teared off together with its three pins. Seems like the only choice is to remove the PBC lacquer and attempt to solder those extra-thin cooper threads. Unless there are alternative pins somewhere...

**Comment by SamanthaRenamon** [comment]
> Yeah, need to fix the R button myself. R button box teared off together with its three pins. Seems like the only choice is to remove the PBC lacquer and attempt to solder those extra-thin cooper threads. Unless there are alternative pins somewhere...
> ![изображение](https://user-images.githubusercontent.com/6737986/67845524-ac48a000-fb10-11e9-9c73-0db277f859f5.png)
> ![изображение](https://user-images.githubusercontent.com/6737986/67845579-c4202400-fb10-11e9-895b-c914e0ab3a0f.png)

I acctually fix it

**Comment by ratijas** [comment]
@SamanthaRenamon Could you be a little bit more specific, please? I still haven't fixed mine, but figured out a lot in the process of trying.

**Comment by jul666** [comment]
Hey !
I've break the same button.
There is an alternate pinout possibility
![joy_con_right_R_button_pin-out](https://user-images.githubusercontent.com/35293005/69904023-62511500-136f-11ea-879a-609419225ef5.jpg)

**Comment by numselli** [comment]
I have this same issue how did you fix it![image](https://user-images.githubusercontent.com/58607232/156901462-417722e5-d7ec-496f-91c1-debbc857abd8.jpeg)

**Comment by jul666** [comment]
![IMG_20220305_190030](https://user-images.githubusercontent.com/35293005/156903774-0222027f-a0b0-49af-aa81-6a98d3d82a73.jpg)
i've fixed mine by using ratijas solution.
- removing lacquer on the track
- soldering the button in the good position with left and right pad
- soldering tiny cooper wire on the track to the button

**Comment by numselli** [comment]
Did you use hot air or an iron?

**Comment by numselli** [comment]
Also is this how the traces connect?![image](https://user-images.githubusercontent.com/58607232/156948635-1d9a10a8-17b3-44ab-a335-b50f84856f70.jpeg)
